.github/workflows/end_to_end.yml

```diff
@@ -2,8 +2,13 @@ name: End to end
 
 on:
   push:
-    branches: [ master, extend-end-to-end ] #<-- replace extend-end-to-end by the name of the agency/publisher
-
+    branches: [ master ]
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - '**.md'
```

Comment:
Do we want to run these workflows if the modified files are related to Gradle, Docker, Git or the license? As it is right now, modifying `.gitignore` or `Dockerfile` would trigger the 3 end-to-end workflows, and I'm not sure this is the expected behaviour. 

If we do not want that, I'd advocate to go with the notation `paths` instead, which will trigger the workflows only if a file following the paths pattern is modified:
```
    paths:
      - 'core/**'
      - 'docs/**'
      - 'main/**'
      - 'processor/**'
      - 'scripts/**'
```

Reply:
I agree with @maximearmstrong that it would be good to avoid running the tests on files that definitely won't change the build, but I'd prefer to specify excluded files instead of included files. Otherwise when we refactor we need to remember to change this file for the tests to actually run. For example, if we create a new module `newModule`, we might forget to modify this GitHub file, and then the tests don't run (and we might not realize it).

So I'd suggest specifying additional specific file types or specific files to ignore here instead.

Reply:
Good point :) Thanks @barbeau

Reply:
Good catch! https://github.com/MobilityData/gtfs-validator/pull/1007/commits/3590a3ccfc52a348fd95002675b7f364e98cb66d disables end to end tests when the following files are modified: 
- `Dockerfile`
- `.gitignore`
- `LICENSE`

> Do we want to run these workflows if the modified files are related to Gradle [...]

> [...] it would be good to avoid running the tests on files that definitely **won't change the build,** 

⬆️ For this reason, I suggest keeping the end to end tests in case gradle build files are modified